src/main/java/io/vlingo/wire/fdx/bidirectional/netty/server/NettyInboundHandler.java

```diff
@@ -28,6 +20,12 @@
 import io.vlingo.wire.channel.ResponseSenderChannel;
 import io.vlingo.wire.message.ConsumerByteBuffer;
 import io.vlingo.wire.message.ConsumerByteBufferPool;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
 
 final class NettyInboundHandler extends ChannelInboundHandlerAdapter implements ResponseSenderChannel {
   private final static Logger logger = LoggerFactory.getLogger(NettyInboundHandler.class);
```

Comment:
@alexguzun We should find a way to use the `io.vlingo.actors.Logger` in all our Netty code to remain async and unified with the rest of the platform.